dev/TeachingTip/TeachingTip_rs1_themeresources.xaml

```diff
@@ -2,36 +2,62 @@
-    xmlns:local="using:Microsoft.UI.Xaml.Controls">
+    xmlns:local="using:Microsoft.UI.Xaml.Controls"
+    xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)"
+    xmlns:contract7NotPresent="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract,7)">
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Default">
-            <StaticResource x:Key="TeachingTipBorderBrush" ResourceKey="SurfaceStrokeColorDefaultBrush"/>
-            <SolidColorBrush x:Key="TeachingTipTopHighlightBrush" Color="#0DFFFFFF"/>
-            <StaticResource x:Key="TeachingTipTransientBackground" ResourceKey="SystemControlPageBackgroundChromeLowBrush"/>
-            <StaticResource x:Key="TeachingTipForegroundBrush" ResourceKey="TextFillColorPrimaryBrush"/>
-            <StaticResource x:Key="TeachingTipBackgroundBrush" ResourceKey="SystemControlPageBackgroundChromeLowBrush"/>
-            <StaticResource x:Key="TeachingTipTitleForegroundBrush" ResourceKey="TextFillColorPrimaryBrush"/>
-            <StaticResource x:Key="TeachingTipSubtitleForegroundBrush" ResourceKey="TextFillColorPrimaryBrush"/>
+            <StaticResource x:Key="TeachingTipAlternateCloseButtonBackground"                                 ResourceKey="SubtleFillColorTransparentBrush" />
+            <StaticResource x:Key="TeachingTipAlternateCloseButtonBackgroundPointerOver"                      ResourceKey="SubtleFillColorSecondaryBrush" />
+            <StaticResource x:Key="TeachingTipAlternateCloseButtonBackgroundPressed"                          ResourceKey="SubtleFillColorTertiaryBrush" />
+            <StaticResource x:Key="TeachingTipAlternateCloseButtonBackgroundDisabled"                         ResourceKey="SubtleFillColorTransparentBrush" />
+            <StaticResource x:Key="TeachingTipAlternateCloseButtonForeground"                                 ResourceKey="TextFillColorPrimaryBrush" />
+            <StaticResource x:Key="TeachingTipAlternateCloseButtonForegroundPointerOver"                      ResourceKey="TextFillColorPrimaryBrush" />
+            <StaticResource x:Key="TeachingTipAlternateCloseButtonForegroundPressed"                          ResourceKey="TextFillColorSecondaryBrush"/>
+            <StaticResource x:Key="TeachingTipAlternateCloseButtonForegroundDisabled"                         ResourceKey="TextFillColorDisabledBrush" />
+            <StaticResource x:Key="TeachingTipBorderBrush"                                                    ResourceKey="SurfaceStrokeColorDefaultBrush"/>
+            <SolidColorBrush x:Key="TeachingTipTopHighlightBrush"                                             Color="#0DFFFFFF"/>
+            <StaticResource x:Key="TeachingTipTransientBackground"                                            ResourceKey="SystemControlPageBackgroundChromeLowBrush"/>
```

Comment:
SystemControlPageBackgroundChromeLowBrush is legacy brush. I would think it should be a new WinUI 2.6 brush?

Reply:
I didn't modify this brush in this PR:) other visual updates will be addressed in a separate PR